lib/client/ClientRenderer.js

```diff
@@ -3,18 +3,21 @@
 var ClientRenderer = {
 
     render: function(layout, view, requestId) {
-        var title = view.getTitle ? view.getTitle() : null;
-        layout.setTitle(title);
+        var shouldRenderTitle = requestId !== 1;
 
-        var shouldRenderMetaTags = requestId === 1 || layout.updateMetatagsOnClientRender;
+        if (shouldRenderTitle) {
+            var title = view.getTitle ? view.getTitle() : null;
+            layout.renderTitle(title);
+        }
+
+        var shouldRenderMetaTags = requestId !== 1 && layout.updateMetatagsOnClientRender;
```

Comment:
What was here before was very incorrect. We should NEVER re-render the metatags on initial page load. The server takes care of that for us.